BrainCloudClient/tests/TestGlobalApp.cs

```diff
@@ -17,5 +17,36 @@ public void TestBrainCloudGlobalApp()
 
             tr.Run();
         }
+        
+        [Test]
+        public void TestGlobalAppReadSelectedProperties()
+        {
+            TestResult tr = new TestResult(_bc);
+            string[] propertyIds =
+            {
+                "prop1"
+            };
+
+            _bc.GlobalAppService.ReadSelectedProperties(propertyIds,tr.ApiSuccess, tr.ApiError);
+
+            tr.Run();
+        }
+        
+        [Test]
+        public void TestGlobalAppReadPropertiesInCategories()
+        {
+            TestResult tr = new TestResult(_bc);
+            string[] categories =
+            {
+                "Test",
+                "pack",
+                "prop1",
```

Comment:
this is not a category. Its test. Pack shouldnt even be there, that's a SynkD thing.
I made a mistake on the portal and have 2 pack categories. I changed them back to test.

Reply:
![image](https://user-images.githubusercontent.com/6390816/134960462-9ef8b277-6760-4c1e-a753-83e8dbd6c5de.png)